website/management/commands/base_lab_seeder.py

```
@@ -0,0 +1,55 @@
+from website.models import Labs, TaskContent, Tasks
+
+
+class LabSeederCommand(BaseCommand):
+    lab_name = None  # Override in subclass
+    tasks_data = []  # Override in subclass
+
+    def handle(self, *args, **kwargs):
+        try:
+            lab = Labs.objects.get(name=self.lab_name)
+        except Labs.DoesNotExist:
+            raise CommandError(f"{self.lab_name} lab not found. Please run create_initial_tasks first.")
```

Ruff (0.14.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)